src/python/gem5/components/boards/se_binary_workload.py

```diff
@@ -105,13 +106,17 @@ def set_se_binary_workload(
         process.executable = binary_path
         process.cmd = [binary_path] + arguments
         if stdin_file is not None:
-            process.input = stdin_file.get_local_path()
+            if isinstance(stdin_file, Path):
+                process.input = FileResource(stdin_file).get_local_path()
+            process.input = FileResource(Path(stdin_file)).get_local_path()
         if stdout_file is not None:
-            process.output = stdout_file.as_posix()
+            if isinstance(stdout_file, Path):
+                process.output = stdout_file.as_posix()
+            process.output = Path(stdout_file).as_posix()
```

Comment:
Almost exactly the same comments here. This should've always been a Path because that's its type.

Also you're overriding the value you set on 114 on 115.

Reply:
@kunpai : Ditto here.